Add Translation Guideline (#948)
pypa/packaging.python.org: source/contribute.rst

:doc:`example specification-style document <specifications/core-metadata>`.

Translations
============

We use `Weblate`__ to manage translations of this project.
Please visit the `packaging.python.org`__ project on Weblate to contribute.

If you are experiencing issues while you are working on translations,
please open an issue on `Github`__.

> .. Note:: Any translations of this project should follow `reStructuredText syntax`__.

Reviewer: Normally, admonitions (and other RST directives) are lowercase.
```suggestion
.. note::

   Any translations of this project should follow `reStructuredText syntax`__.
```

.. __: https://weblate.org/
.. __: https://hosted.weblate.org/projects/pypa/packaging-python-org/
.. __: https://github.com/pypa/packaging.python.org/issues

> .. __: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reviewer: Please, make the references named. It's hard to maintain things that are hard to map and easy to break links by accidentally injecting some reference in the middle.

Adding a language
-----------------

If your language is not listed on `Weblate`__, click the button
"Start new translation" at the bottom of the languages list and add
the language you want to translate.

> .. __: https://hosted.weblate.org/projects/pypa/packaging-python-org/

Reviewer: This link could be reused.

Following reStructuredText syntax
---------------------------------

If you are not familiar with reStructuredText (RST) syntax, please read `this guide`__
before translating on Weblate.

Example:

Wrong: Translate the following text directly::

  `some ref`_ -> `TRANSLATED TEXT HERE`_

Right: Translate the following text with your own language and add reference to the 
original language::

>   `some ref`_ -> `TRANSLATED TEXT HERE <some ref>`_

Reviewer: Maybe we should follow the style of right/wrong from the bottom of this document?

:doc:`example specification-style document <specifications/core-metadata>`.

Translations
============

We use `Weblate`_ to manage translations of this project.
Please visit the `packaging.python.org`_ project on Weblate to contribute.

If you are experiencing issues while you are working on translations,
please open an issue on `Github`_.

> .. note::

Reviewer: I guess it's more of a tip than a note:
```suggestion
.. tip::
```

   Any translations of this project should follow `reStructuredText syntax`_.

.. _Weblate: https://weblate.org/
.. _packaging.python.org: https://hosted.weblate.org/projects/pypa/packaging-python-org/
.. _Github: https://github.com/pypa/packaging.python.org/issues
.. _reStructuredText syntax: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Adding a language
-----------------

If your language is not listed on `packaging.python.org`_, click the button

> "Start new translation" at the bottom of the languages list and add

Reviewer: ```suggestion
:guilabel:`Start new translation` at the bottom of the language list and add
```

the language you want to translate.

Following reStructuredText syntax
---------------------------------

If you are not familiar with reStructuredText (RST) syntax, please read `this guide`_
before translating on Weblate.

**Do not translate the text in reference directly**

  When translating the text in reference, please do not translate them directly.

  | Wrong: Translate the following text directly:

>   .. code-block::

Reviewer: ```suggestion
  .. code-block:: rst
```

      `some ref`_ -> `TRANSLATED TEXT HERE`_

  | Right: Translate the following text with your own language and add the original reference:

>   .. code-block::

Reviewer: ```suggestion
  .. code-block:: rst
```

If you are experiencing issues while you are working on translations,
please open an issue on `Github`_.

.. note::

   Any translations of this project should follow `reStructuredText syntax`_.

.. _Weblate: https://weblate.org/
.. _packaging.python.org: https://hosted.weblate.org/projects/pypa/packaging-python-org/
.. _Github: https://github.com/pypa/packaging.python.org/issues
.. _reStructuredText syntax: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Adding a language
-----------------

If your language is not listed on `packaging.python.org`_, click the button
"Start new translation" at the bottom of the languages list and add
the language you want to translate.

Following reStructuredText syntax
---------------------------------

If you are not familiar with reStructuredText (RST) syntax, please read `this guide`_
before translating on Weblate.

**Do not translate the text in reference directly**

  When translating the text in reference, please do not translate them directly.

  | Wrong: Translate the following text directly:

  .. code-block::

      `some ref`_ -> `TRANSLATED TEXT HERE`_

>   | Right: Translate the following text with your own language and add reference to the 
>     original language:

Reviewer: How about
```suggestion
  | Right: Translate the following text with your own language and add the original reference:
```

Author: Sounds better.
